pkg/agent/utils.go

```diff
@@ -586,3 +586,25 @@ func addFeatureGateString(featureGates string, key string, value bool) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// ParseCSE parses the raw VMSS CSE output
+func ParseCSE(message string) (datamodel.VMSSInstanceViewCSEStatus, error) {
```

Comment:
ParseCSEMessage or ParseCSEOutput ?

Reply:
based on Tony's comment https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp/pullrequest/4175240?_a=files&path=%2Fresourceprovider%2Fserver%2Fvendor%2Fgithub.com%2FAzure%2Fagentbaker%2Fpkg%2Fagent%2Fbaker.go&discussionId=73055024, only ParseCSE should be fine.

Reply:
no I mean ParseCSEMessage or ParseCSEOutput

In my comment I highlighted only the "ParseVMSSCSE" part and say just ParseCSE. Will try to be thorough in future comments.